src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs

```diff
@@ -278,9 +278,18 @@ public PagerShapes(IStringLocalizer<PagerShapes> localizer)
             {
                 for (var p = firstPage; p <= lastPage; p++)
                 {
+                    if (p == 1)
```

Comment:
It's a little misleading because "firstPage" isn't the absolute first page, it's the first number link that is visible. Here the firstPage would be 2.

![image](https://user-images.githubusercontent.com/68876423/166457125-12dc6773-88ee-43ac-b715-995132501140.png)

Reply:
> But if the first page is 2 then it is the first page. Not sure if this is a common use case but that's what it should do whatever the first pagenum it is. Though I accepted the PR because it is a really rare use case.

Sorry, I'm not following. We want to remove pagenum query parameter only when the page number == 1. Even if 2 is the first page visible we wouldn't want to remove the pagenum parameter because then it would link to the wrong page.

Reply:
Not necessarily if the URL is /something and that it is intended to point to a pagenum=2 from server-side code.
In that case : 

`/something == /something?pagenum=2`

and 

`/something?pagenum=1` doesn't exists.